tools/container/hue/supervisor-files/hue_loglistener_template

```diff
@@ -0,0 +1,18 @@
+[program:hue_loglistener]
+directory=${HUE_HOME}
+command="${HUE_HOME}/desktop/core/src/desktop/loglistener.py"
```

Comment:
Does it also need the path to python3?

Reply:
One the docker only python3.8 is installed.

Reply:
oh, I didn't realize that it is in the `container` directory. Sounds good.